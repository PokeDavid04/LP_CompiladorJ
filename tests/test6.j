NB. Filtros y selecciones
nums =: 1 2 3 4 5 6 7 8
filtro =: 1 0 1 0 1 0 1 0
filtro # nums
