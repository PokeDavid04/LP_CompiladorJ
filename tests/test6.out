
square =: <asignada> 
1 4 9 16 


add5 =: <asignada> 
5 6 7 8 9 


mod2 =: <asignada> 
0 1 0 1 0 1 

mod4 =: <asignada> 
0 1 2 3 0 1 2 3 


isZero =: <asignada> 
1 0 1 0 1 

greaterThan3 =: <asignada> 
0 0 1 1 


eq0 =: <asignada> 
div3 =: <asignada> 
div3Test =: <asignada> 
1 0 0 1 0 0 1 0 0 


inc =: <asignada> 
double =: <asignada> 
incDouble =: <asignada> 
12 22 32 


sumRange =: <asignada> 
10 

prodRange =: <asignada> 
120 


nums =: <asignada> 
mask =: <asignada> 
1 3 5 


getSecond =: <asignada> 
20 


parell =: <asignada> 
1 0 1 0 1 0 


cube =: <asignada> 
9 27 81 


createAndCount =: <asignada> 
7 


complexChain =: <asignada> 
30 


double2 =: <asignada> 
10 20 30 

